README.md

```diff
@@ -1 +1,90 @@
-# book-store
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Awesome books ](#-awesome-books-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Bookstore CMS <a name="about-project"></a>
+
+**Calculator** in this project, i create a Bookstore using React.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML | CSS | JS | ES6 | React
+
+### Key Features <a name="key-features"></a>
+
+- **[Add Books]**
+- **[Delete Books]**
+- **[React Router Dom]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [LIVE DEMO UNAVAILABLE]
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+A GitHub | A code editor | A web browser
+
+### Install
+
+Install all project dependencies by running the command below
+
+  `$ npm install`
+  
+And run `npm start` to launch the project.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  `git clone https://github.com/huzaifakhan0308/book-store.git`
+
+### Usage
+
+To run the project, run `npm start` to run the project
+
+## 👥 Author <a name="author"></a>
+
+👤 **Huzaifa khan**
+
+- GitHub: [@huzaifakhan0308](https://github.com/huzaifakhan0308)
+- Twitter: [@huzaifakhan0308](https://twitter.com/home?lang=en)
+- LinkedIn: [huzaifa khan](https://www.linkedin.com/in/huzaifa-khan-938140256/)
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Feel free to check [issues page](https://github.com/huzaifakhan0308/book-store/issues).
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```

Comment:
- [ ] Please make sure to include Future Features, including 1-3 potential features to be added.